spec/liquid_tags/unified_embed/tag_spec.rb

```diff
@@ -58,12 +58,18 @@
     end.to raise_error(StandardError, "URL provided may have a typo or error; please check and try again")
   end
 
-  it "raises an error when no link-matching class is found" do
-    link = "https://takeonrules.com/about"
+  it "calls OpenGraphTag when no link-matching class is found", vcr: true do
+    link = "https://takeonrules.com/about/"
 
-    expect do
-      stub_head_request(link)
-      Liquid::Template.parse("{% embed #{link} %}")
-    end.to raise_error(StandardError, "Embeds for this URL are not supported")
+    allow(OpenGraphTag).to receive(:new).and_call_original
+
+    VCR.use_cassette("takeonrules_fetch") do
```

Comment:
Thanks for the hat tip!  This looks like a nice and straight forward spec that demonstrates how we're connecting these concepts.